modules/gitlab/module/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     gitlab = {
       source  = "gitlabhq/gitlab"
-      version = ">=3.5.0"
+      version = ">=0.7.0"
```

Comment:
Is this intentional? It seems like we're opening it up to earlier versions of the gitlab provider.

Reply:
its unnecessary - inherited code form jenkins branch